### PR TITLE
disable chronos

### PR DESCRIPTION
**Pull request by ringabout**
EDIT(timotheecour): refs https://github.com/nim-lang/Nim/issues/17130

### CODE REVIEW

File: testament/important_packages.nim [hunk 1]
```diff
@@ -25,8 +25,9 @@ pkg1 "cascade"
 pkg1 "cello"
 pkg1 "chroma"
 pkg1 "chronicles", "nim c -o:chr -r chronicles.nim"
-when not defined(osx): # testdatagram.nim(560, 54): Check failed
-  pkg1 "chronos", "nim c -r -d:release tests/testall"
+# when not defined(osx): # testdatagram.nim(560, 54): Check failed
+#   pkg1 "chronos", "nim c -r -d:release tests/testall"
+  # pending https://github.com/status-im/nim-chronos/issues/159
```
Comment (ringabout) [inline testament/important_packages.nim]:
```suggestion
  # pending https://github.com/nim-lang/Nim/issues/17130
```

**Review by timotheecour — APPROVED** [review]

**Comment by timotheecour** [comment]
merging to unblock PR's but wouldn't this work?
```nim
`pkg1 "chronos", "nim c -r -d:release tests/testall", useHead = false`
```

untested but seems to address this:
> Please make release tags, otherwise it is more likely to break Nim CI which always requests latest chronos version)

**Comment by ringabout** [comment]
without release tags, which version it will USE?

**Comment by timotheecour** [comment]
indeed, https://github.com/status-im/nim-chronos has no tags, so it is a bug with nim-chronos (or shouldn't be in important_packages)

we should add a note on top of important_packages.nim explaining that each package listed should have release tags along with rationale

**Comment by timotheecour** [comment]
=> PR https://github.com/nim-lang/Nim/pull/17134

**Pull request merged**